tools/cli/main.c

```diff
@@ -162,6 +162,7 @@ int main(int argc, char **argv)
         {
             TDNFCliShowUsage();
         }
+        dwError = ERROR_TDNF_CLI_NO_SUCH_CMD;
     }
```

Comment:
we do not need to BAIL_ON_CLI_ERROR after setting dwError?

Reply:
Thank you, you are correct. I'll fix it.